.gitignore

```diff
@@ -1,3 +1,5 @@
+/bin/
```

Comment:
This shouldn't go into .gitignore, as you usually shouldn't use the project root as `$GOPATH$`.

Reply:
Ah, I didn't know that. So what should be used as $GOPATH instead?